### ISSUE TITLE
'class -> "..." on form overrides bootstrap classes

### ISSUE THREAD

**Issue by mkurz** [adrianhurt/play-bootstrap3#25]
This form...

```scala
@b3.form(routes.Application.login) {
}
```
... renders to following HTML (here the vertical field constructor was used, but it does not matter which field constructor you use):
```html
<form action="/" method="GET" class="form-vertical" role="form">
</form>
```

But now when adding a class to the form, the bootstrap `form-*` classes are missing (in my example it is `form-vertical`):
```scala
@b3.form(routes.Application.login, 'class -> "myclass") {
}
```
becomes:
```html
<form action="/" method="GET" class="myclass" role="form">
</form>
```
As you can see only `myclass` gets used, but `form-vertical` is gone! Right now I have to add the `form-vertical` css class by my own - which IMHO is very bad. If I change a `@b3.inline.form` to a `@b3.horizontal.form` I do not want to think about to also change the css class...

I suggest `@b3.form.*` (or `@b3.vertical.form`, `@b3.inline.form`, etc., whatever)  **always should add the corresponding bootstrap form class**, no matter if we define `'class -> "..."`.

**Comment by adrianhurt** [comment]
Yeah, I think you're right.

**Comment by adrianhurt** [comment]
Now it's fixed in 0.4.2-SNAPSHOT. Thanks!

**Comment by mkurz** [comment]
@adrianhurt Thanks! Will try it soon.
